### ISSUE TITLE
Small intestinal bacterial overgrowth

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6187]
**Preferred term label:**
Small intestinal bacterial overgrowth

**Synonyms**
SIBO, Small intestinal bacterial overgrowth syndrome

**Definition (free text, please give PubMed ID)**
An increase in the number and/or alteration in the type of bacteria in the upper gastrointestinal tract. 
PMID: 20572300

**Parent term (use hpo.jax.org/app)**
Abnormality of the small intestine

**Comment by pnrobinson** [comment]
Term added as HP:0033163
